tika/server.go

```diff
@@ -100,7 +100,11 @@ func NewServer(jar, port string) (*Server, error) {
 	if err != nil {
 		return nil, fmt.Errorf("invalid port %q: %v", s.port, err)
 	}
+
 	s.url = u.String()
+
+	s.Javaprops = make(map[string]string)
```

Comment:
Mind moving this above to the declaration of `s`? Does it need to be pre-initialized?